src/app/content/components/Content.tsx

```diff
@@ -27,31 +27,35 @@ import { wrapperPadding } from './Wrapper';
 
 // tslint:disable-next-line:variable-name
 const Background = styled.div`
+@media screen {
```

Comment:
I can indent these properly once the PR is approved. Not indenting them makes it easier to pull in updates from `master`